File: controle-demandas-api/src/main/java/com/controle/demandas/api/dtoDemandas/DemandaSearchDTO.java
package com.controle.demandas.api.dtoDemandas;

import lombok.*;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class DemandaSearchDTO {

    private Long id;
    private String titulo;
    private String descricao;
    private String status;
    private String cpfCidadao;
}

File: AppFrontEnd/src/app/dtos/dto-demandas/demanda-search.dto.ts
export class DemandaSearchDTO {
  id!: number;
  titulo!: string;
  descricao!: string;
  status!: "Aberta" | "Em Andamento" | "Concluída" | "Cancelada" | "Não Concluída";
  cpfCidadao!: string;
}
